### ISSUE TITLE
✅ ♿ 🐞 Bug Report: Nav menu widget is not keyboard accessible

### ISSUE THREAD

**Issue by swinggraphics** [elementor/elementor#20583]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Title says it all. This problem exists on every Elementor site, even elementor.com.

### Steps to reproduce

Use the keyboard to highlight a mobile menu toggle (`.elementor-menu-toggle`). Press the return key. Nothing happens.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

Clean install. 3.8.1 & 3.8.2. This is a universal problem. System info is not necessary.

**Comment by swinggraphics** [comment]
Fix:
````
$( '.elementor-menu-toggle' ).on( 'keypress', function(event){
	if ( 13 === event.which ) $(this).click();
} );
````

**Comment by swinggraphics** [comment]
@nicholaszein This is not specific to my site. You can see this problem on your own website—elementor.com. System info is completely irrelevant.

**Comment by nicholaszein** [comment]
Hello @swinggraphics,

thanks for the feedback! 🙏 

we need the System Info to be able to reproduce your issue and track it. Please provide the System Info as requested so we can move this forward.

Thanks

**Comment by swinggraphics** [comment]
> we need the System Info to be able to reproduce your issue and track it. Please provide the System Info as requested so we can move this forward.

Then I suggest you get the System Info from elementor.com.

**Comment by alfredo49** [comment]
Same issue Toggle menu isnt accesible is impossible open it

**Comment by nicholaszein** [comment]
Thank you very much @alfredo49! 🙏

This information is crucial for the team, even if it works elsewhere. This is why we ask for the System Info beforehand.

You're a real gentleman! 🧐

**Comment by alfredo49** [comment]
Any updates about this issue? is very important for accesibility this issue was fixed soon :-(

**Comment by rami-elementor** [comment]
@alfredo49 

We have verified the issue. A new task was created to fix the bug.

**Comment by askwpgirl** [comment]
Please let us know when this will be fixed. It's been an issue for many years and should be a priority.

**Comment by rami-elementor** [comment]
Fixed in 3.13.0

The version expected to be released on May 9.

**Comment by askwpgirl** [comment]
Thank you!

**Comment by nicholaszein** [comment]
Hello, everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.13.0**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
